source/_patterns/01-molecules/article-type/article-type.twig

```diff
@@ -0,0 +1,7 @@
+<ul class="article__type">
```

Comment:
'Article type' is one of the items that will appear in this list rather than the list itself. The eLife equivalent pattern is [Meta](https://ui-patterns.elifesciences.org/?p=molecules-meta), but I don't think that's a good name.

Reply:
To be discussed. Maybe Article types as could be more than one?